lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -1071,6 +1071,42 @@
     }
   }
 
+  // cohort discussions interface.
+  .cohort-discussions-nav {
+
+    .cohort-coursewide-discussions-form {
+
+      .form-actions {
+        padding-top: ($baseline/2);
+      }
+    }
+
+    .category-title,
+    .topic-name,
+    .all-inline-discussions,
+    .always_cohort_inline_discussions {
+      padding-left: ($baseline/2);
+    }
+
+    .always_cohort_inline_discussions {
+      padding-top: ($baseline/2);
+    }
+
+    .category-item {
+      padding-top: ($baseline/2);
+    }
+
+    .cohorted-text {
+      color: $link-color;
+    }
+
+    .discussions-wrapper {
+      @extend %ui-no-list;
+      padding: 0 ($baseline/2);
+    }
+  }
+
+
```

Comment:
For .wrapper-tabs &.subcategories, I would suggest using padding: 0 ($baseline*1.5); (this lines up the checkbox visually with the start of the text from the parent's li element text string.